src/router/routes/deploy.js

```diff
@@ -0,0 +1,132 @@
+//
+// SecureImage
+//
+// Copyright © 2018 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Created by Jason Leach on 2018-01-10.
+//
+
+/* eslint-env es6 */
+
+'use strict';
+
+import * as minio from 'minio';
+import url from 'url';
+import util from 'util';
+import request from 'request-promise-native';
+import { Router } from 'express';
+import config from '../../config';
+import { logger } from '../../libs/logger';
+import {
+  asyncMiddleware,
+  errorWithCode,
+} from '../../libs/utils';
+import {
```

Comment:
createBucketIfRequired, bucketExists, isExpired are now in @bcgov/nodejs-common-util.